### DISCUSSION TITLE
Increasing spacing between two surfaces

### DISCUSSION THREAD

**Discussion by KodalAnwesha** [gempy-project/gempy#949] [Q&A] [answered]
Hello,
I am creating models based on real data. In my scenario, the spacing between two surfaces is almost 3 or 4 meters. So the two surfaces in the model are not clearly separable. 
Can you tell me a solution so that this spacing can be magnified?
I am attaching the photos here.
![model1](https://github.com/user-attachments/assets/3f99f8f2-0587-421d-b7d6-0e4a253e6ffb)
![model2](https://github.com/user-attachments/assets/32af88e0-1698-49fa-8fbd-0e1175c63b32)
![model3](https://github.com/user-attachments/assets/b7965e08-4e32-4295-86d0-bd13ca59c253)

Thank you!

**Comment by NilsChudalla** [comment]
Dear @KodalAnwesha, 

to me it looks like the model predicts your data quite precisely. So, I am struggling to see if there is anything wrong with your model. I take it, that you are looking for a visual fix to your problem. How about this one: Use the "ve" (vertical exaggeration) argument in your plots. ve=2 would indicate, that 1 meter in the vertical direction is displayed twice as long as 1m in horizontal direction (purely visual effect). 
Cheers
Nils

**Reply by NilsChudalla** [reply]
Excuse me, seems like you already did use "ve". Could you please specify your problem further? Because interpolation wise it looks fine to me

**Reply by KodalAnwesha** [reply]
Hello, 
My actual problem is, as these polyline_0 and polyline_4 surfaces are very close to each other they are not properly separable visually. I want to increase the visual vertical distance in between them without affecting their original vertical distance. More like a zoomed version. 

I tried decreasing the total z dimension if it can be zoomed, but it didn't work. Can you please suggest me a way. Because I will have to insert more such surfaces in this model. It will look very crowded.

And can you explain more about this ve? I don't think I had used it in my plot.

**Reply by KodalAnwesha** [reply]
I tried the ve argument in code. But it didn't work.

**Reply by NilsChudalla** [reply, marked as answer]
I am using gempy version 2024.1.4

If I do the basic 2d plot, without additional arguments <code>gpv.plot_2d(geo_model)</code> it looks like this:
<img src="https://github.com/user-attachments/assets/76d0f503-d3a1-4e24-bbbc-b17723cfbf22" width=40%>

and if I add the vertical exaggeration <code>gpv.plot_2d(geo_model, ve=5)</code> it looks like this:
<img src="https://github.com/user-attachments/assets/a5accf1f-5486-464d-af11-f7528a9e501d" width=40%>

Vertical exaggeration is a common concept in mapping if you are not familiar with it and there should be plenty of resources online. 

Was this reply helpful? :)

**Reply by KodalAnwesha** [reply]
Yes yes, I got your point.
But in my model this thing is not working.
Thanks though!!

**Reply by NilsChudalla** [reply]
If you send me the code snippet, that is not working, with the corresponding error message, I might be able to help. This information is not sufficient to solve your problem.

**Reply by KodalAnwesha** [reply]
Yes, now the error is solved and the ve command is working. 
Thank you very much for your help :)
